docs/blog/posts/anyscale.md

```diff
@@ -73,7 +73,7 @@ resp = client.chat.completions.create(
     response_model=UserDetails,
 )
 print(resp)
->>> name='Jason' age=20
+# >>> name='Jason' age=20
```

Comment:
Personal preference, but it makes it easier to copy, paste, and run the whole thing.